tools/devservices_healthcheck.py

```diff
@@ -8,7 +8,7 @@
 def run_cmd(
     args: list[str],
     *,
-    retries: int = 3,
+    retries: int = 6,
```

Comment:
lol - could maybe add a clickhouse healthcheck to complete before starting snuba?

```
snuba is not ready: <urlopen error [Errno 111] Connection refused>
snuba is not ready: <urlopen error [Errno 111] Connection refused>
snuba is not ready: <urlopen error [Errno 111] Connection refused>
snuba is not ready: <urlopen error [Errno 111] Connection refused>
snuba is not ready: <urlopen error [Errno 111] Connection refused>
snuba is not ready: timed out
```

Reply:
yeah probably :/ I'm amazed at how slow it starts in CI :(